frontend/src/components/pages/Home/Home.js

```diff
@@ -5,13 +5,24 @@ class Home extends React.Component {
     render = () => {
         return (
             <>
-                <ul>
-                    <li><Link to="/create-room">Create room</Link></li>
-                    <li><Link to="/join-room">Join room</Link></li>
-                </ul>
+                <div className="label-title">LE QUIZ
```

Comment:
Why a div if it's just for the main title ? Plus maybe we gonna use the logo we have to talk about that and we have special component for this purpose (watch my comment in style.css)